entrypoint_debian.sh

```diff
@@ -46,20 +46,57 @@ fi
 # Copy initial files to host volume
 case ${OPENHAB_VERSION} in
   1.8.3)
-      if [ -z "$(ls -A "${APPDIR}/configurations")" ]; then
+      if [ -z a"$(ls -A "${APPDIR}/configurations")" ]; then
         # Copy userdata dir for version 1.8.3
         echo "No configuration found... initializing."
         cp -av "${APPDIR}/configurations.dist/." "${APPDIR}/configurations/"
       fi
     ;;
   2.0.0|2.1.0|2.2.0|2.3.0-snapshot)
       # Initialize empty host volumes
-      if [ -z "$(ls -A "${APPDIR}/userdata")" ]; then
+      if [ -za"$(ls -A "${APPDIR}/userdata")" ]; then
```

Comment:
Why add the 'a' option here? It's not added for the alpine image and it doesn't seem to solve warnings when the dir does not exist.

Reply:
see above